### ISSUE TITLE
Security fix for Moodle 4.1

### ISSUE THREAD

**Issue by HoffmannP** [studentquiz/moodle-mod_studentquiz#504]
Reading the release notes to 5.3.0 there seems to be a security issues. Unfortunately the fix is only backported to MOODLE_STABLE_3x but not to the current only available Moodle LTS  Version 4.1 - or am I missing something?

**Comment by timhunt** [comment]
You are correct. Becuase of chagnes in the Moodle core question bank, there have had to be significant changes in StudentQuiz, and we only have the resources to develop the main branch.

If you are able to backport the fixes, then please do, and let me know, and I will release a 4.1-compatible branch. (We did not backport anythign the the 3x branch. That is no longer supported.)

**Comment by HoffmannP** [comment]
Oh sorry, I assumed the comment
```
Moodle 4.3 and 4.4 - if you need a version compatible with earlier versions of Moodle, look on the MOODLE_3X_STABLE branch
```
in the recently changed `README.md` would still be applicable.

**Comment by timhunt** [comment]
You are right. That bit in the readme is not accurate, and we should fix it. Sorry.
